deploy/Dockerfile

```
@@ -0,0 +1,101 @@
+    API_HOST=0.0.0.0 \
+    API_PORT=8882 \
+    PGSERVE_EMBEDDED=false \
+    NATS_MANAGED=false \
+    HOME=/home/bun
+WORKDIR /app
+
+# ffmpeg: the audio processor (packages/media-processing/src/processors/audio.ts)
+# normalizes/transcodes WhatsApp opus voice notes before handing them to the
+# transcription provider. Installed here (as root, before USER bun) so audio STT
+# works out of the box; the slim base omits it.
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)